pymystrom/switch.py

```diff
@@ -48,16 +48,28 @@ async def get_state(self) -> None:
         """Get the details from the switch/plug."""
         url = URL(self.uri).join(URL("report"))
         response = await request(self, uri=url)
-        self._consumption = response["power"]
-        self._consumedWs = response["Ws"]
+        try:
+            self._consumption = response["power"]
+        except KeyError:
+            self._consumption = None
+        try:
+            self._consumedWs = response["Ws"]
+        except KeyError:
+            self._consumedWs = None
         self._state = response["relay"]
         try:
             self._temperature = response["temperature"]
         except KeyError:
             self._temperature = None
 
-        url = URL(self.uri).join(URL("info.json"))
+        """ try the new API """
```

Comment:
Please make this a comment.

Reply:
Yes, I've changed lines 65 and 69 to comments, see my last commit.

Reply:
I have tested it with the example-switch.py on both my switches. Is there another proper way to write tests? I'm rather new to python programming.

Reply:
Unit tests for a standard switch and a switch zero.

Reply:
Guess that's something for later...